vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -6212,6 +6212,16 @@ unit:GM-PER-DeciM3
   rdfs:label "GM PER DeciM3" ;
   skos:prefLabel "gram per decimetre cubed" ;
 .
+unit:GM-PER-GM
```

Comment:
I think this should be treated according to our discussion, namely with a quantitykind of DimensionlessRatio, and with a contextualquantitykind of MassRatio. Let me know if you'd like me to make these changes.

Reply:
The example you are showing, Steve, is GM-PER-DeciM3, which is not a MassRatio but a MassDensity, isn't it?

Reply:
My comment is attached to unit:GM-PER-GM.

Reply:
![image](https://user-images.githubusercontent.com/1130189/82074685-fdc11280-968f-11ea-86de-3fb36c52854b.png)